### DISCUSSION TITLE
想问问大家TCGA基因之间相关性那个星号怎么加不上来了

### DISCUSSION THREAD

**Discussion by shaonianabin666** [ShixiangWang/self-study#121] [Q&A] [answered]
代码如下
vis_gene_cor_cancer(
  Gene1 = "基因1",
  Gene2 = "基因2",
  data_type1 = "mRNA",
  data_type2 = "mRNA",
  purity_adj = FALSE,
  cancer_choose = "TGCT",
  use_regline = TRUE,
  cor_method = "spearman",
  use_all = FALSE,
  alpha = 0.5,
  color = "#000000",
  opt_pancan = .opt_pancan
)
我向往里面加入  Show.P.value = TRUE,Show.P.label = TRUE,加不进去，想看P值，求助各位大佬

**Comment by lishensuo** [comment, marked as answer]
默认是可以P值与星号的，如下代码所示，不需要Show.P.value与Show.P.label参数。

```r
vis_gene_cor_cancer(
  Gene1 = "CSF1R",
  Gene2 = "JAK3")
```
![image](https://github.com/user-attachments/assets/6cd46f81-c534-4519-b152-d30c8e53b3a5)

**Reply by ShixiangWang** [reply]
@lishensuo 本身的各个选项对应的动作逻辑是正确的么？如果不对的话纠正一下

**Reply by lishensuo** [reply]
`vis_gene_cor_cancer()` 本身的参数都是可以正常使用的
